components/dashboard/src/app/OrgRequiredRoute.tsx

```diff
@@ -5,13 +5,13 @@
  */
 
 import { Redirect, Route } from "react-router";
-import { useCurrentTeam } from "../teams/teams-context";
+import { useCurrentOrg } from "../data/organizations/orgs-query";
 
 // A wrapper for <Route> that redirects to "/" if there is not an organization currently selected
 // Having a check for an active org at the route level allows us to avoid any org-dependant api calls we might make
 // in page level components, and having to check for an org there
 export function OrgRequiredRoute({ component }: any) {
-    const org = useCurrentTeam();
+    const org = useCurrentOrg()?.data;
```

Comment:
I believe we can drop the `?` here since we always return an object from the hook (just data itself may be undefined).